### ISSUE TITLE
Unprocessable Entity

### ISSUE THREAD

**Issue by louiscoetzee** [spatie/laravel-pjax#11]
Hi, I installed the middleware as described in the readme, but in the console on the get request I can see an error: unprocessable entity and then the page redirects.

Please help. Is there something I am missing?

I am using laravel 5.2

**Comment by louiscoetzee** [comment]
Please note, there's no validation error as I am trying to load the index action/view

**Comment by louiscoetzee** [comment]
I am using Postman to try and debug. Looks like it's line Line 87 that is returning the error code 422.
I am not sure why this is happening, it means my content is empty? Why?
Thanks in advance.

**Comment by louiscoetzee** [comment]
`<!DOCTYPE html>
<html>
    <head>
        <meta name="robots" content="noindex,nofollow" />
        <style>
            /* Copyright (c) 2010, Yahoo! Inc. All rights reserved. Code licensed under the BSD License: http://developer.yahoo.com/yui/license.html */
            html{color:#000;background:#FFF;}body,div,dl,dt,dd,ul,ol,li,h1,h2,h3,h4,h5,h6,pre,code,form,fieldset,legend,input,textarea,p,blockquote,th,td{margin:0;padding:0;}table{border-collapse:collapse;border-spacing:0;}fieldset,img{border:0;}address,caption,cite,code,dfn,em,strong,th,var{font-style:normal;font-weight:normal;}li{list-style:none;}caption,th{text-align:left;}h1,h2,h3,h4,h5,h6{font-size:100%;font-weight:normal;}q:before,q:after{content:'';}abbr,acronym{border:0;font-variant:normal;}sup{vertical-align:text-top;}sub{vertical-align:text-bottom;}input,textarea,select{font-family:inherit;font-size:inherit;font-weight:inherit;}input,textarea,select{*font-size:100%;}legend{color:#000;}
            html { background: #eee; padding: 10px }
            img { border: 0; }
            #sf-resetcontent { width:970px; margin:0 auto; }
                        .sf-reset { font: 11px Verdana, Arial, sans-serif; color: #333 }
            .sf-reset .clear { clear:both; height:0; font-size:0; line-height:0; }
            .sf-reset .clear_fix:after { display:block; height:0; clear:both; visibility:hidden; }
            .sf-reset .clear_fix { display:inline-block; }
            .sf-reset * html .clear_fix { height:1%; }
            .sf-reset .clear_fix { display:block; }
            .sf-reset, .sf-reset .block { margin: auto }
            .sf-reset abbr { border-bottom: 1px dotted #000; cursor: help; }
            .sf-reset p { font-size:14px; line-height:20px; color:#868686; padding-bottom:20px }
            .sf-reset strong { font-weight:bold; }
            .sf-reset a { color:#6c6159; cursor: default; }
            .sf-reset a img { border:none; }
            .sf-reset a:hover { text-decoration:underline; }
            .sf-reset em { font-style:italic; }
            .sf-reset h1, .sf-reset h2 { font: 20px Georgia, "Times New Roman", Times, serif }
            .sf-reset .exception_counter { background-color: #fff; color: #333; padding: 6px; float: left; margin-right: 10px; float: left; display: block; }
            .sf-reset .exception_title { margin-left: 3em; margin-bottom: 0.7em; display: block; }
            .sf-reset .exception_message { margin-left: 3em; display: block; }
            .sf-reset .traces li { font-size:12px; padding: 2px 4px; list-style-type:decimal; margin-left:20px; }
            .sf-reset .block { background-color:#FFFFFF; padding:10px 28px; margin-bottom:20px;
                -webkit-border-bottom-right-radius: 16px;
                -webkit-border-bottom-left-radius: 16px;
                -moz-border-radius-bottomright: 16px;
                -moz-border-radius-bottomleft: 16px;
                border-bottom-right-radius: 16px;
                border-bottom-left-radius: 16px;
                border-bottom:1px solid #ccc;
                border-right:1px solid #ccc;
                border-left:1px solid #ccc;
            }
            .sf-reset .block_exception { background-color:#ddd; color: #333; padding:20px;
                -webkit-border-top-left-radius: 16px;
                -webkit-border-top-right-radius: 16px;
                -moz-border-radius-topleft: 16px;
                -moz-border-radius-topright: 16px;
                border-top-left-radius: 16px;
                border-top-right-radius: 16px;
                border-top:1px solid #ccc;
                border-right:1px solid #ccc;
                border-left:1px solid #ccc;
                overflow: hidden;
                word-wrap: break-word;
            }
            .sf-reset a { background:none; color:#868686; text-decoration:none; }
            .sf-reset a:hover { background:none; color:#313131; text-decoration:underline; }
            .sf-reset ol { padding: 10px 0; }
            .sf-reset h1 { background-color:#FFFFFF; padding: 15px 28px; margin-bottom: 20px;
                -webkit-border-radius: 10px;
                -moz-border-radius: 10px;
                border-radius: 10px;
                border: 1px solid #ccc;
            }
        </style>
    
    </head>
    <body>
        <div id="sf-resetcontent" class="sf-reset">
            <h1>Whoops, looks like something went wrong.</h1>
            <h2 class="block_exception clear_fix">
                <span class="exception_counter">1/1</span>
                <span class="exception_title">
                    <abbr title="Symfony\Component\HttpKernel\Exception\HttpException">HttpException</abbr> in 
                    <a title="/Users/Louis/Projects/recruit360/vendor/laravel/framework/src/Illuminate/Foundation/Application.php line 905" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">Application.php line 905</a>:
                </span>
                <span class="exception_message"></span>
                        
            </h2>
            <div class="block">
                <ol class="traces list_exception">
                    <li> in 
                        <a title="/Users/Louis/Projects/recruit360/vendor/laravel/framework/src/Illuminate/Foundation/Application.php line 905" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">Application.php line 905</a>
                    </li>
                    <li>at 
                        <abbr title="Illuminate\Foundation\Application">Application</abbr>->abort('422', '', 
                        <em>array</em>()) in 
                        <a title="/Users/Louis/Projects/recruit360/vendor/laravel/framework/src/Illuminate/Foundation/helpers.php line 30" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">helpers.php line 30</a>
                    </li>
                    <li>at 
                        <abbr title=""></abbr>abort('422') in 
                        <a title="/Users/Louis/Projects/recruit360/vendor/spatie/laravel-pjax/src/Middleware/FilterIfPjax.php line 87" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">FilterIfPjax.php line 87</a>
                    </li>
                    <li>at 
                        <abbr title="Spatie\Pjax\Middleware\FilterIfPjax">FilterIfPjax</abbr>->fetchContainer(
                        <em>object</em>(
                        <abbr title="Symfony\Component\DomCrawler\Crawler">Crawler</abbr>), '#listDetails') in 
                        <a title="/Users/Louis/Projects/recruit360/vendor/spatie/laravel-pjax/src/Middleware/FilterIfPjax.php line 54" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">FilterIfPjax.php line 54</a>
                    </li>
                    <li>at 
                        <abbr title="Spatie\Pjax\Middleware\FilterIfPjax">FilterIfPjax</abbr>->filterResponse(
                        <em>object</em>(
                        <abbr title="Illuminate\Http\Response">Response</abbr>), '#listDetails') in 
                        <a title="/Users/Louis/Projects/recruit360/vendor/spatie/laravel-pjax/src/Middleware/FilterIfPjax.php line 35" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">FilterIfPjax.php line 35</a>
                    </li>
                    <li>at 
                        <abbr title="Spatie\Pjax\Middleware\FilterIfPjax">FilterIfPjax</abbr>->handle(
                        <em>object</em>(
                        <abbr title="Illuminate\Http\Request">Request</abbr>), 
                        <em>object</em>(
                        <abbr title="Closure">Closure</abbr>))
                    </li>
                    <li>at 
                        <abbr title=""></abbr>call_user_func_array(
                        <em>array</em>(
                        <em>object</em>(
                        <abbr title="Spatie\Pjax\Middleware\FilterIfPjax">FilterIfPjax</abbr>), 'handle'), 
                        <em>array</em>(
                        <em>object</em>(
                        <abbr title="Illuminate\Http\Request">Request</abbr>), 
                        <em>object</em>(
                        <abbr title="Closure">Closure</abbr>))) in 
                        <a title="/Users/Louis/Projects/recruit360/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php line 124" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">Pipeline.php line 124</a>
                    </li>
                    <li>at 
                        <abbr title="Illuminate\Pipeline\Pipeline">Pipeline</abbr>->Illuminate\Pipeline\{closure}(
                        <em>object</em>(
                        <abbr title="Illuminate\Http\Request">Request</abbr>))
                    </li>
                    <li>at 
                        <abbr title=""></abbr>call_user_func(
                        <em>object</em>(
                        <abbr title="Closure">Closure</abbr>), 
                        <em>object</em>(
                        <abbr title="Illuminate\Http\Request">Request</abbr>)) in 
                        <a title="/Users/Louis/Projects/recruit360/vendor/laravel/framework/src/Illuminate/Routing/Pipeline.php line 32" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">Pipeline.php line 32</a>
                    </li>
                    <li>at 
                        <abbr title="Illuminate\Routing\Pipeline">Pipeline</abbr>->Illuminate\Routing\{closure}(
                        <em>object</em>(
                        <abbr title="Illuminate\Http\Request">Request</abbr>))
                    </li>
                    <li>at 
                        <abbr title=""></abbr>call_user_func(
                        <em>object</em>(
                        <abbr title="Closure">Closure</abbr>), 
                        <em>object</em>(
                        <abbr title="Illuminate\Http\Request">Request</abbr>)) in 
                        <a title="/Users/Louis/Projects/recruit360/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php line 103" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">Pipeline.php line 103</a>
                    </li>
                    <li>at 
                        <abbr title="Illuminate\Pipeline\Pipeline">Pipeline</abbr>->then(
                        <em>object</em>(
                        <abbr title="Closure">Closure</abbr>)) in 
                        <a title="/Users/Louis/Projects/recruit360/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php line 132" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">Kernel.php line 132</a>
                    </li>
                    <li>at 
                        <abbr title="Illuminate\Foundation\Http\Kernel">Kernel</abbr>->sendRequestThroughRouter(
                        <em>object</em>(
                        <abbr title="Illuminate\Http\Request">Request</abbr>)) in 
                        <a title="/Users/Louis/Projects/recruit360/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php line 99" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">Kernel.php line 99</a>
                    </li>
                    <li>at 
                        <abbr title="Illuminate\Foundation\Http\Kernel">Kernel</abbr>->handle(
                        <em>object</em>(
                        <abbr title="Illuminate\Http\Request">Request</abbr>)) in 
                        <a title="/Users/Louis/Projects/recruit360/public/index.php line 54" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">index.php line 54</a>
                    </li>
                    <li>at 
                        <abbr title=""></abbr>require_once('/Users/Louis/Projects/recruit360/public/index.php') in 
                        <a title="/Users/Louis/Projects/recruit360/server.php line 21" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">server.php line 21</a>
                    </li>
    
                </ol>
            </div>

            
        </div>
    
    </body>
</html>`

**Comment by louiscoetzee** [comment]
Okay, I fixed it. Make sure the view you are loading has the target id for example id="listDetails"

**Comment by CupOfTea696** [comment]
If you figured this out, you may want to close this issue :)
